Agents/Xamarin.Interactive/Collections/ImmutableList.cs

```diff
@@ -59,6 +59,14 @@ public ImmutableList<T> InsertRange (int index, IEnumerable<T> items)
             return new ImmutableList<T> (newItems);
         }
 
+        public int FindLastIndex (Predicate<T> match)
+        {
```

Comment:
Throw `ArgumentNullException` for `match`

Reply:
Done.